### ISSUE TITLE
CVAT-SDK Python: Consistent naming assignee_id, assignee for task/job

### ISSUE THREAD

**Issue by niclaswue** [opencv/cvat#6255]
I find it very confusing that a `job` has the field `assignee` and a `task` can only be assigned via `assignee_id`, as mentioned here: https://github.com/opencv/cvat/issues/5951#issuecomment-1497060253

The `task` object also has a property called `assignee` which made it even more confusing for me. Is there a good reason for this difference in naming? It took me multiple hours to find this bug, it should be changed, made synonymous or at least be better documented.

**Comment by realtimshady1** [comment]
From what I understand and how I use it, a task can have multiple jobs, like different chunks (job) of a video (task) are assigned to different people. The people doing the labelling job can't see jobs that are not assigned to them, nor the umbrella task. This way, we keep the labelling more independent and contribution more distributed.

**Comment by niclaswue** [comment]
Yes, that's right, but it doesn't explain why the two classes have a different naming schema. In the UI, the elements are similar for a job and a task.
